client/post-editor/post-editor.jsx

```diff
@@ -713,9 +713,6 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
-		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
One other thing to think about. Currently, since the editor is not reduxified, saving a post is called using the old Actions but when we reduxify everything, we would probably call the `savePost` action which could create the same bug. So maybe the fix for this would be to clear the edits when triggering the request instead of when receiving the response. The downside of this approach is when the save fails.